Arduino/victoria_teensy_ros_node/RosParamHelper.h

```diff
@@ -0,0 +1,80 @@
+#ifndef ROS_PARAM_HELPER_h
```

Comment:
Please refer to the Google Style Guide https://google.github.io/styleguide/cppguide.html#The__define_Guard
(But the ROS style guide first which is refers to Google on this issue http://wiki.ros.org/CppStyleGuide)

Let's say it should be
`#ifndef VICTORIA_TEENSY_ROS_NODE_ROS_PARAM_HELPER_H_`

I think that would be right.